packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -898,12 +908,12 @@ export function createLodestarMetrics(
         }),
         gossipInsertOutcome: register.counter<{insertOutcome: InsertOutcome}>({
           name: "lodestar_oppool_attestation_pool_gossip_insert_outcome_total",
-          help: "Total number of InsertOutcome as a result of adding a gossip attestation in a pool",
+          help: "Total number of InsertOutcome as a result of adding a single attestation from gossip to the pool",
           labelNames: ["insertOutcome"],
         }),
         apiInsertOutcome: register.counter<{insertOutcome: InsertOutcome}>({
           name: "lodestar_oppool_attestation_pool_api_insert_outcome_total",
-          help: "Total number of InsertOutcome as a result of adding an attestation from api in a pool",
+          help: "Total number of InsertOutcome as a result of adding a single attestation from api in the pool",
```

Comment:
```suggestion
          help: "Total number of InsertOutcome as a result of adding a single attestation from api to the pool",
```

Reply:
ah good catch, I tried a few variants to make it clearer, looks like I missed that